Libraries/Alert/NativeAlertManager.js

```diff
@@ -22,6 +23,11 @@ export type Args = {|
   cancelButtonKey?: string,
   destructiveButtonKey?: string,
   keyboardType?: string,
+  // [TODO(macOS ISS#2323203)
+  defaultInputs?: DefaultInputsArray,
+  modal?: ?boolean,
+  critical?: ?boolean,
```

Comment:
The macOS Alert styles are critical, informational, and warning so I don't think toggling on a "critical" as a bool correctly encapsulates the three options available

Reply:
AFAIK Flow has no enums, so then this should probably be a string union?

Reply:
Would require changes to the `RCTAlertManager.mm`, as it currently takes `critical` as a boolean.